diff --git a/src/Amqp.csproj b/src/Amqp.csproj
index b8b44f2..9dbb0c8 100644
--- a/src/Amqp.csproj
+++ b/src/Amqp.csproj
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <AssemblyName>Amqp.Net</AssemblyName>
     <PackageId>AmqpNetLite</PackageId>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
